runtime/polkadot/src/lib.rs

```diff
@@ -821,12 +822,28 @@ impl pallet_bounties::Config for Runtime {
 	type BountyUpdatePeriod = BountyUpdatePeriod;
 	type BountyCuratorDeposit = BountyCuratorDeposit;
 	type BountyValueMinimum = BountyValueMinimum;
-	type ChildBountyManager = ();
+	type ChildBountyManager = ChildBounties;
 	type DataDepositPerByte = DataDepositPerByte;
 	type MaximumReasonLength = MaximumReasonLength;
 	type WeightInfo = weights::pallet_bounties::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const MaxActiveChildBountyCount: u32 = 100;
+	pub const ChildBountyValueMinimum: Balance = BountyValueMinimum::get() / 10;
```

Comment:
Same as for Kusama

This is too high IMO. There may be use cases where the child bounty rewards need to be quite low ([example](https://docs.google.com/document/d/1V-39ktdgpvATPpV1cfI8iZlS8_3oBDAHdlcMDd_3jpo/edit?usp=sharing)) or at least disproportionally low to the total bounty.

I propose a fixed min of 5 * `DOLLARS` (which I believe is half the min of the main bounty).

Reply:
will be addressed in follow up PR